include/swift/SIL/SILBridging.h

```diff
@@ -804,9 +804,19 @@ struct BridgedSILTypeArray {
   BridgedSILTypeArray(llvm::ArrayRef<swift::SILType> silTypes)
       : typeArray(silTypes) {}
 
+  BridgedSILTypeArray(BridgedArrayRef typeArray)
+      : typeArray(typeArray) {}
+
   llvm::ArrayRef<swift::SILType> unbridged() const {
     return typeArray.unbridged<swift::SILType>();
   }
+
```

Comment:
👍 

Just for my information, why does BridgedValueArray need `getValues()` but BridgeSILTypeArray doesn't? Is it because Value is a protocol and bridged Value are actually existentials, but SILType is a class?